src/components/Chooser/Chooser.test.js

```diff
@@ -5,14 +5,18 @@ import { render } from '@testing-library/react';
 import { shallow } from 'enzyme';
 
 describe('Chooser', () => {
-  let wrapper;
+  let wrapper, instance;
   const mockProjects = [
     {id: 1, name: 'test1'},
     {id: 2, name: 'test2'}
   ];
 
-  it('Should match the snapshot', () => {
+  beforeEach(() => {
     wrapper = shallow(<Chooser projects={mockProjects} />);
+    instance = wrapper.instance();
```

Comment:
Do we need to use instance here?  I thought that instance was only used for class-based components?